### PR TITLE
replace "First name" and "Surname" input fields with a single "Full Name" field

### PR DESCRIPTION
**Pull request by sahilpatel2712**
# ✨ Codu Pull Request 💻

Fixes issue #1080 

## Pull Request details

- replace "First name" and "Surname" input fields with a single "Full Name" field
- add Zod validation for the "Full Name" input field.

## Any Breaking changes

'None'

## Associated Screenshots
Screenshot 2024-10-07 210429

### CODE REVIEW

**Review by John-Paul-Larkin — CHANGES_REQUESTED** [review]
Looks great @sahilpatel2712 
One last thing. Could you please drop the firstName and surname columns from the database schema since we’re using fullname instead. You can modify the migration accordingly.
Thanks

File: app/(app)/alpha/additional-details/page.tsx [hunk 1]
```diff
@@ -13,8 +13,7 @@ export default async function Page() {
   const details = await db.query.user.findFirst({
     columns: {
       username: true,
-      firstName: true,
-      surname: true,
+      name: true,
```
[the remark this answers is not in the document]
Comment (John-Paul-Larkin) [reply app/(app)/alpha/additional-details/page.tsx:16]: This can be safely ignored, since this alpha/settings page is legacy and will be deleted.

**Comment by sahilpatel2712** [comment]
sure...

**Review by John-Paul-Larkin — DISMISSED** [review]
Great work, thanks @sahilpatel2712

**Comment by sahilpatel2712** [comment]
> Great work, thanks @sahilpatel2712

You're welcome!

**Comment by NiallJoeMaher** [comment]
Just some prettier issues. Run `npm run prettier:fix` and it should solve it! Great work.

**Comment by sahilpatel2712** [comment]
> Just some prettier issues. Run `npm run prettier:fix` and it should solve it! Great work.

done..

**Review by NiallJoeMaher — APPROVED** [review]
🌮 🦖

**Pull request merged**